### ISSUE TITLE
Blank home page

### ISSUE THREAD

**Issue by bdecarne** [mobz/elasticsearch-head#92]
Hi,

On my 0.90.6 installation with only 1 node and 1 index i've got a blank home page. There is a javavascript error in app.js line 3038 :

```
Uncaught TypeError: Cannot read property 'attributes' of undefined 
```

Content of the "node" object : 

```
node: Object
     cluster: undefined
     master_node: false
     name: "Unassigned"
     routings: Array[1]
     stats: undefined
```

**Comment by huchenme** [comment]
this issue appears again in Chrome

**Comment by philipskokoh** [comment]
@huchenme: which elasticsearch version are you using?

**Comment by huchenme** [comment]
2.1.1

**Comment by philipskokoh** [comment]
If you open Chrome developer tools (Ctrl+Shift+I), do you see any error messages?

**Comment by huchenme** [comment]
@philipskokoh I see `Uncaught TypeError: this.statEl.text(...).css is not a function` error

**Comment by philipskokoh** [comment]
Hmm.. I tested using elasticsearch 2.1.1. It works well on my side.

**Comment by philipskokoh** [comment]
@huchenme: I am unable to replicate your problem. Any more messages you see before that error?

**Comment by huchenme** [comment]
@philipskokoh no, it is working in Safari, but in Chrome it is blank, console only have that message

**Comment by jthiesse** [comment]
@philipskokoh I have the exact same behavior as @huchenme. Strange error in Chrome but loads fine in Safari. Version 2.1.1

**Comment by andyskipper** [comment]
I have this issue in Chrome as well - ES version 2.2.0, fresh install 20 minutes ago.

<img width="960" alt="screen shot 2016-02-21 at 13 27 09" src="https://cloud.githubusercontent.com/assets/381270/13202772/de1d2e04-d89e-11e5-8d7e-80c6207b3e95.png">

**Comment by andyskipper** [comment]
Just found the cause of this - if you have a Chrome extension such as Proper Blocker, it adds a script tag to the body. The i18n library used for translations mistakes this extra script tag for the one defining its dataset-langs attribute, hence can't load the translation strings. 

The i18n library included is pretty fragile, and I'd suggest future versions look at alternatives.

**TL;DR** Disable Chrome extensions such as adblockers temporarily, as they may be inserting script tags which kill the localisation.

**Comment by mobz** [comment]
@andyskipper I wonder if you could try running es with this patch to see if it fixes the script injection problem.
[fix.92.patch.txt](https://github.com/mobz/elasticsearch-head/files/140579/fix.92.patch.txt)

( you can apply the patch with `git apply fix.92.patch.txt` in the root folder of elasticsearch-head
